[2025-02-06T10:36:52.958+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-06T10:36:52.970+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_que_faire_a_paris.extract_data manual__2025-02-06T10:36:51.763826+00:00 [queued]>
[2025-02-06T10:36:52.976+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_que_faire_a_paris.extract_data manual__2025-02-06T10:36:51.763826+00:00 [queued]>
[2025-02-06T10:36:52.977+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-06T10:36:52.986+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_data> on 2025-02-06 10:36:51.763826+00:00
[2025-02-06T10:36:52.992+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=5200) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-06T10:36:52.993+0000] {standard_task_runner.py:72} INFO - Started process 5202 to run task
[2025-02-06T10:36:52.993+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_que_faire_a_paris', 'extract_data', 'manual__2025-02-06T10:36:51.763826+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/dag_que_faire_a_paris.py', '--cfg-path', '/tmp/tmp2f_66xav']
[2025-02-06T10:36:52.994+0000] {standard_task_runner.py:105} INFO - Job 102: Subtask extract_data
[2025-02-06T10:36:53.024+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_que_faire_a_paris.extract_data manual__2025-02-06T10:36:51.763826+00:00 [running]> on host b176106a6d6f
[2025-02-06T10:36:53.277+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_que_faire_a_paris' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-06T10:36:51.763826+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-06T10:36:51.763826+00:00'
[2025-02-06T10:36:53.277+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-06T10:36:53.849+0000] {dag_que_faire_a_paris.py:40} INFO - âœ… Extraction completed successfully!
[2025-02-06T10:36:53.851+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-06T10:36:53.858+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-06T10:36:53.859+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_que_faire_a_paris, task_id=extract_data, run_id=manual__2025-02-06T10:36:51.763826+00:00, execution_date=20250206T103651, start_date=20250206T103652, end_date=20250206T103653
[2025-02-06T10:36:53.889+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-06T10:36:53.906+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-06T10:36:53.909+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
